tools/plotjuggler/juggle.py

```diff
@@ -46,6 +47,13 @@ def install():
       tar.extractall(path=INSTALL_DIR)
 
 
+def check_plotjuggler_version():
+  proc = subprocess.Popen([PLOTJUGGLER_BIN, "-v"], stdout=subprocess.PIPE)
+  (out, _) = proc.communicate()
+  version = out.decode("utf-8").split(" ")[1]
+  return version >= MINIMUM_PLOTJUGGLER_VERSION
```

Comment:
these are strings. it'll work properly if you split into a tuple

Reply:
Works as strings as well, but can do tuples if prefered.

Reply:
Not always:

```
"0.8.9" > "0.8.10"
Out[126]: True
```